Bundle/WidgetBundle/Builder/WidgetFormBuilder.php

```diff
@@ -34,7 +34,7 @@ public function __construct(Container $container)
      *
      * @return string
      */
-    public function renderNewForm($form, $widget, $slot, View $view, $entity = null)
+    public function renderNewForm($form, $widget, $slot, View $view, $entity = null, $quantum = null)
```

Comment:
I will check this update my work with this work but it's sure that it does not make it easier.